src/types.rs

```diff
@@ -15,6 +17,12 @@ pub struct Coordinate<T>
     pub y: T,
 }
 
+impl<T: Float> From<(T, T)> for Coordinate<T> {
```

Comment:
After sleeping on this, I think you make a good point (heh) about `Point::new`. As long as we have that constructor looking like `Point::new(3, 7)` which implicitly is `(x, y)`, it doesn't seem much worse to make a `From` impl for `(T, T)`. For the time being, I think this new `impl` is fine and maybe in the future we should state in the module level documentation that all coordinate pairs in rust-geo will be `x, y`.